ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_C.yml

```diff
@@ -13,7 +13,6 @@ Tenant_C:
       update_source: lo0
       bfd: false
       ebgp_multihop: 3
-      set_ipv4_next_hop: 1.1.1.1
```

Comment:
So here we are removing these - I can see these in our eos_designs documentation but I suppose it never worked? Should we log an issue to fix our documentation ?

```
    bgp_peer_groups:
      - [...]
        set_ipv4_next_hop: < IPv4_address >   
```

the migrated code for network services only considers the set_ipv4_next_hop for the bgp_peers and not the peer_groups (as it was before in the template):

https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/route_maps.py#L32

Reply:
Before the python PR, we relayed everything blindly, even these invalid keys, which are not supported in eos_cli_config_gen, so no configuration would be created.
We have two options:
- Remove the key from the documentation
- Implement the key in eos_designs to create the correct eos_cli_config_gen config (with route_maps and everything).

Reply:
I would for now remove the key from documentation as it has never worked - it can be then implemented when / if required I would say - thoughts?

Reply:
Let's just remove it in this PR

Reply:
fixed in [6744175](https://github.com/aristanetworks/ansible-avd/pull/2105/commits/67441755df0e25e6b8f978dc5d11fc694cc50c9c)